README.md

```diff
@@ -24,25 +17,89 @@ Change directory to the package under development. Note: an integration is a spe
 
 ```bash
 cd integrations
-cd package/my-integration
+cd package/my-package
 ```
 
-Run the `help` command and see available actions:
+Run the `help` command and see available commands:
 
 ```bash
 elastic-package help
 ```
 
-## GitHub authorization
+## Commands
+
+`elastic-package` currently offers the commands listed below. 
+
+Some commands have a _global context_, meaning that they can be executed from anywhere and they will have the 
+same result. Other commands have a _package context_; these must be executed from somewhere under a package's
+root folder and they will operate on the contents of that package.
+
+For more details on a specific command, run `elastic-package help <command>`.
+
+### `elastic-package help`
+
+Use this command to get a listing of all commands available under `elastic-package` and a brief
+description of what each command does.
+
+_Context: global_
+
+### `elastic-package version`
+
+Use this command to print the version of `elastic-package` that you have installed. This is
+especially useful when reporting bugs.
+
+_Context: global_
+
+### `elastic-package build`
+
+Use this command to build a package. TODO: what is the purpose of building a package — when would
+a package developer want/need to run this command?
+
+_Context: package_
+
+### `elastic-package lint`
+
+Use this command to validate the contents of a package.
```

Comment:
... against schema files defined by package-spec repository.

Reply:
Referenced the package spec in 51c497473dffecb4d6163351d23d6b2e5bcb5e25.